### ISSUE TITLE
Unknown crash at startup

### ISSUE THREAD

**Issue by TheIrishStud** [paulevsGitch/BetterEnd#380]
https://pastebin.com/psYvH28G

You told me I downloaded the wrong version before. Hope this helps further bug reporting.

**Comment by paulevsGitch** [comment]
There are no things that can point issue in these logs (related to BE), can you please post a crashlog too?

**Comment by TheIrishStud** [comment]
Apologies for not being able to find and provide the previously crashed Better End test so I re-added it to my mod list and managed to load into Minecraft before crashing as I clicked to load my 1.18 world after backing it up. I do believe however that the previous crash was caused by an incompatibility with Terralith 2.0. Here is both the latest log text as well as the crash text posted beneath the dashed lines separating the 2.

[crash-2022-01-02_09.46.28-server.txt](https://github.com/paulevsGitch/BetterEnd/files/7799550/crash-2022-01-02_09.46.28-server.txt)

[latest.log](https://github.com/paulevsGitch/BetterEnd/files/7799551/latest.log)

Apologies for not using pastebin. Something supposedly offensive prevented me from copy/pasting publicly.
